lib/fix-google-html.js

```diff
@@ -144,6 +150,20 @@ function convertInlineStylesToElements(node) {
   visitParents(node, n => isStyled(n) && !isBlock(n), (node, parents) => {
     const style = resolveNodeStyle(node, parents);
 
+    // normalize the tag name
+    if (
+      (
+        (node.tagName === 'b' || node.tagName === 'strong')
+          && (style['font-weight'] === 'normal' || style['font-weight'] === '400')
+      ) ||
+      (
+        (node.tagName === 'i' || node.tagName === 'em') &&
+        style['font-style'] === 'normal'
+      )
+    ) {
+      node.tagName = 'span';
+    }
```

Comment:
Is there a case you're hitting where Google Docs is outputting a `<b>/<strong>/<i>/<em>` tag and then setting the weight to not be bold/italic? I don't think I've seen this; curious about when it happens.

Reply:
I removed this since I could not find any clear examples of when/where this was needed in practice, and in the abstract, I think it's pretty debatable. Should the resulting markdown match the semantic info in the HTML or the styling in place? A `<strong>` could be restyled to not be bold, but instead be big, colored, underlined, or something else to show emphasis. It's not obvious that just because a `<strong>` is not bold it should not equate to `**text**` in markdown.

Reply:
I added that because the root element of the paste content is a bold element for some reason. However, that problem was resolved when I switched back to letting the browser handle the paste (instead of manually updating the innerHTML with the clipboard content on paste), so this is indeed obsolete. And I think you are right, the text might be bolder by default, and then this would break, so good call on removing it 👍